src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -612,10 +616,10 @@ bool operator==(const Frustum& lhs, const Frustum& rhs) {
          lhs.half_opening_angle_ == rhs.half_opening_angle_ and
          lhs.one_over_tan_half_opening_angle_ ==
          rhs.one_over_tan_half_opening_angle_ and
-         std::isnan(lhs.zeta_distribution_value_) ==
-             std::isnan(rhs.zeta_distribution_value_) and
-         ((std::isnan(lhs.zeta_distribution_value_) and
-           std::isnan(rhs.zeta_distribution_value_)) or
+         ((std::isnan(lhs.zeta_distribution_value_) or
```

Comment:
why this change? I found the previous easier to understand

Reply:
Oh for some reason I thought the logic in the other one was causing the difference, so I tried this out. I can revert it back if you want.

Reply:
Reverted it back